### PR TITLE
Add digital certificate

### PR DESCRIPTION
**Pull request by root21cuib**

### CODE REVIEW

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:8]:
```suggestion
Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely. 

Certificates authenticate the user through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservices](https://glossary.cncf.io/microservices/) app. 
```

Comment (CathPag) [reply content/en/digital_certificate.md:8]: Edited the text a bit. Same content but worded slightly differently. This feels like a bit too much information: "through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions." It will probably raise more questions than answering them. Should we skip this? Maybe this is enough?

> Certificates authenticate users through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservices](https://glossary.cncf.io/microservices/) app.

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:11]: This section here is far too technical. Please remember that we are trying to make this accessible to non-technical people as well. TCP/IP, OSI network model, security handshakes, layers between apps and networks...these are all things that aren't commonly understood. Could please try to explain this without all these industry terms?

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:14]:
```suggestion
Digital certificates are the most practical option for securing data online. Certificates encrypt internal and external communications to prevent outsiders and phishers from stealing sensitive information. They provide the same encryption quality for small to large businesses alike. 

Digital certificates are highly scalable and can even be used to secure personal devices. Institutions known as public trusted certificate authorities handle the issuance of digital certificates, ensuring they are reliable. Certificate management can be easily centralized through managed public key infrastructure platforms. 
```

Comment (CathPag) [reply content/en/digital_certificate.md:14]: Mostly edits here again (content didn't change). I did remove "By ultimately encrypting your company website and digitally signing your documents and emails, you are projecting a positive reputation to your clients," as this seems to be providing advice or judging those who don't do that. I think we need to keep it highly factual. 

Also, do we need "They provide the same encryption quality for small to large businesses alike.?" Why would the quality change depending on your size? After all, it's the same technology. I think we should skip that sentence. What do you think?

**Review by CathPag — COMMENTED** [review]
Great submission @root21cuib! It will require review from a more technical approver but sections 1 and 3 sounded pretty good to me but did need some edits. I also left two comments.

Section 2 is too technical. We'll need to simplify that. Feel free to reach out if you have any questions.

File: content/en/digital_certificate.md
Comment (root21cuib) [reply content/en/digital_certificate.md]: Thanks very much @CathPag  for the feedback. I revised my work and made some updates to it

**Comment by root21cuib** [comment]
@CathPag  Please can you review my work. I added a new commit where my made updates

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:9]: Looks like you might have missed this comment in my previews feedback:

This feels like a bit too much information: "through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions." It will probably raise more questions than answering them. Should we skip this? Maybe this is enough?

> Certificates authenticate users through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservices](https://glossary.cncf.io/microservices/) app.

Comment (CathPag) [reply content/en/digital_certificate.md:9]: We need a line between the two paragraphs

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:12]:
```suggestion
Networks, such as the internet, are inherently insecure. Communication is managed through protocols (rules and policies to process data), but these are generic and designed to focus on robust communication rather than secure transactions. There is no centralized mechanism to secure communication. Consequently, each application must have its authentication strategy. Rather than building a generic, reusable security mechanism, applications relied on proprietary or embedded techniques for shuffling account information across the network for validation.
```

Comment (CathPag) [reply content/en/digital_certificate.md:12]: I removed the first sentence because, ideally, in the problem it addresses section, we don't even mention the solution. The remaining changes are stylistic.

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:16]: Please add a line between the paragraphs.

**Review by CathPag — COMMENTED** [review]
Thanks, @root21cuib! The middle section is way simpler now :) I made a few edits but will need a technical approver to review the actual content. Please clean it up and I'll send it over to the next approver.

File: content/en/digital_certificate.md
Comment (root21cuib) [reply content/en/digital_certificate.md:12]: @CathPag thankyou for the feedback. I am fixing it right away

**Review by CathPag — COMMENTED** [review]
Looks great! Now let's have a technical review look at the content. There are a few things I'm not super familiar with. @iamNoah1, please have a look when you get a chance.

File: content/en/digital_certificate.md
Comment (JasonMorgan) [inline content/en/digital_certificate.md:10]: I would like to drop line 10. This goes into a specific use case and isn't sufficiently general.

File: content/en/digital_certificate.md
Comment (JasonMorgan) [inline content/en/digital_certificate.md:12]: I think this needs to be reworked. While authenticating a user is one possible use for certificates there are many others including the much more common use of providing the underlying identity and encryption mechanism for TLS.

This entire section needs to be rewritten in a way that is more broadly applicable. 

That being said this is a great effort and I really appreciate your work.

File: content/en/digital_certificate.md
Comment (JasonMorgan) [inline content/en/digital_certificate.md:2]: Why are we calling this a digital certificate instead of just certificate? I don't necessarily object but the title was less clear to me than it should be. @iamNoah1 @seokho-son @CathPag I think this item demonstrates why we need to add a metadata field for alternative names. An alternate name section would let us provide other common names for a particular entry.

File: content/en/digital_certificate.md
Comment (JasonMorgan) [inline content/en/digital_certificate.md:18]: I think you're on the right track but the how it helps needs to be simplified. It might be worth it to talk about TLS more broadly or even add a reference to a good definition of TLS as certificates are one aspect of TLS. I'd also like to see a link to a good definition of PKI. To be clear I do **not** think the glossary should include either term.

**Review by JasonMorgan — CHANGES_REQUESTED** [review]
Great work, sorry for all the requested changes. I believe you're on the right track.

**Review by iamNoah1 — COMMENTED** [review]
friendly ping @root21cuib

File: content/en/digital_certificate.md
Comment (root21cuib) [reply content/en/digital_certificate.md:18]: @JasonMorgan  thankyou for the feedbacks. concerning this section, should I completely change the explannation concerning the How it helps ?

**Comment by CathPag** [comment]
@root21cuib, are you still interested in working on this? If not, I'll like to make the term available to someone else. Thank you!

File: content/en/digital_certificate.md
Comment (CathPag) [inline content/en/digital_certificate.md:10]: I don't think this example works. Certificates can't tell me if the website I'm on is what it claims it is — the internet is full of fraudulent offerings. That aside, Jason is asking to go into the identity and encryption mechanisms for TLS and make it broader so we aren't only focusing on user authentication. 

If this is not something you are familiar with, no worries. Please say so and we'll get someone else to help with that.

**Review by CathPag — COMMENTED** [review]
@root21cuib, added a comment to your update.

File: content/en/digital_certificate.md
Comment (root21cuib) [reply content/en/digital_certificate.md:10]: @CathPag , I am sorry I have lot of struggle to fix what's asked from me. Yes please, I wish for someone to help me with that

Comment (CathPag) [reply content/en/digital_certificate.md:10]: No worries. We'll take it from here :)

**Comment by iamNoah1** [comment]
Abandoned for #527

**Pull request closed without merging**
